### DISCUSSION TITLE
安卓

### DISCUSSION THREAD

**Discussion by MIt-gancm** [BetterAndroid/FlexiUI#2] [General]
什么时候才能吧安卓文档放出来啊
真的非常喜欢这个库

**Comment by fankes** [comment]
> 什么时候才能吧安卓文档放出来啊
> 真的非常喜欢这个库 

社畜中，还在开发，最近可能实在是没空

**Comment by fankes** [comment]
等一下，你是指的哪个库？

**Reply by MIt-gancm** [reply]
ui库   重构的这个   佬加油

**Reply by fankes** [reply]
只能慢慢来了，compose 的目前可以直接用
